lib/manager/mix/extract.js

```diff
@@ -0,0 +1,49 @@
+const { isValid } = require('../../versioning/npm');
+
+module.exports = {
+  extractPackageFile,
+};
+
+function extractPackageFile(content) {
+  logger.debug('mix.extractPackageFile()');
+  const deps = [];
+
+  let lineNumber = 0;
+  for (const line of content.split('\n')) {
+    const pakageMatch = line.match(/{:(\w+),\s*([^:"]+)?:?\s*"([^"]+)".*}/);
```

Comment:
Are you sure that deps are always defined one per line? Do we skip them gracefully if someone uses an unusual multi-line syntax?

Reply:
Also, should we make sure we're inside a `defp deps() do` block before we start parsing lines?